app/client/src/components/shared/HelpTooltip.js

```diff
@@ -83,6 +83,7 @@ function HelpTooltip({ label }) {
         <span aria-hidden>
           <i className="fas fa-question-circle" />
         </span>
+        <span className="sr-only">Information Tooltip</span>
```

Comment:
Can you remove the usage of `data-testid` on line 79 above, since it is no longer used anywhere?

Reply:
I'll go ahead and commit this change up and merge this in, so you can disregard this.